Gemfile.lock

```diff
@@ -1038,8 +1072,9 @@ GEM
     zeitwerk (2.6.13)
 
 PLATFORMS
+  arm64-darwin-22
```

Comment:
This needs to be switched back to `ruby`.

Reply:
```
bundle lock --add-platform ruby
bundle lock --remove-platform arm64-darwin-22
bundle install --redownload
```